### DISCUSSION TITLE
Address bar suggestions not working

### DISCUSSION THREAD

**Discussion by monkeyangst** [searx/searx#2977] [Installation issue: custom]
I've never been able to get my private instance's autocomplete suggestion feature working from the address bar, either in Chrome or Firefox. The other day I decided to spin up a new VM and install Searx using the included install scripts, using Nginx as my HTTP server. I've added nothing but the config files Searx adds by default.

My instance is at http://search.monkeyangst.com/searx/

I've been comparing my setup to that which I can suss out from searx.be -- their suggestions are working just fine, it seems.

Here is _their_ opensearch.xml file:
```
<?xml version="1.0" encoding="utf-8"?>
<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/">
  <ShortName>Searx Belgium</ShortName>
  <Description>a privacy-respecting, hackable metasearch engine</Description>
  <InputEncoding>UTF-8</InputEncoding>
  <Image>https://searx.be/static/themes/oscar/img/favicon.png</Image>
  <LongName>searx metasearch</LongName>
    <Url rel="results" type="text/html" method="post" template="https://searx.be/search">
        <Param name="q" value="{searchTerms}" />
    </Url>
    <Url rel="suggestions" type="application/x-suggestions+json" template="https://searx.be/autocompleter?q={searchTerms}"/>

  <Url type="application/opensearchdescription+xml"
      rel="self"
      template="/opensearch.xml?method=POST&amp;autocomplete=google" />
</OpenSearchDescription>
```

and here is mine:

```
<?xml version="1.0" encoding="utf-8"?>
<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/">
  <ShortName>searx</ShortName>
  <Description>a privacy-respecting, hackable metasearch engine</Description>
  <InputEncoding>UTF-8</InputEncoding>
  <Image>http://search.monkeyangst.com/searx/static/themes/oscar/img/favicon.png</Image>
  <LongName>searx metasearch</LongName>
    <Url rel="results" type="text/html" method="post" template="http://search.monkeyangst.com/searx/search">
        <Param name="q" value="{searchTerms}" />
    </Url>
    <Url rel="suggestions" type="application/x-suggestions+json" template="http://search.monkeyangst.com/searx/autocompleter?q={searchTerms}"/>

  <Url type="application/opensearchdescription+xml"
      rel="self"
      template="/searx/opensearch.xml?method=POST&amp;autocomplete=google" />
</OpenSearchDescription>
```

Apart from the actual addresses, and the use of HTTPS (which I can use if needed), I cannot discern any differences there. So the issue is, to my mind, not caused by that file. So, what else should I be looking at?

**Comment by allendema** [comment]
Try changing the autocompleter because it looks like your wanted autocomplete engine blocks your requests.

 [http://search.monkeyangst.com/searx/search?q=%21go%20too%20many%20requests&categories=none](http://search.monkeyangst.com/searx/search?q=%21go%20too%20many%20requests&categories=none&language=de-DE)

**Reply by monkeyangst** [reply]
That is very interesting. I occasionally have been getting "too many requests" from a regular search, but figured that was a separate issue. I wonder how the Belgium folks get around that, as I see that they, too, are using Google for autocomplete.

**Comment by unixfox** [comment]
You need to set google as a default autocomplete in the settings.yml of Searx.
The line: https://github.com/searx/searx/blob/master/searx/settings.yml#L18

Your GitHub discussion is related to https://github.com/searx/searx/issues/2082
